### PR TITLE
(draft) new API endpoints

### PR DESCRIPTION
**Pull request by groovecoder**
# API endpoit to create outbound calls
# TODOs
1. [ ] Put this behind waffle flag
2. [ ] Check the relay number has remaining call minutes
3. [ ] After the call, deduct the number of minutes of the call

## Screenshot (if applicable)
image

## How to test
1. Sign in to http://127.0.0.1:8000 with an account that has phone level of Relay
4. Go to http://127.0.0.1:8000/api/v1/docs/
5. Expand the `POST /call` endpoint at the top
6. Click "Try it out"
7. Replace `{"to": "string"}` with `{"to": "+12223334444"}` (use a real  number you have access to)
8. Click "Execute"
   * [ ] You should receive a call to your true phone number from your relay number
   * [ ] When you answer, it should say "Dialing +12223334444"
   * [ ] It should connect your call to +12223334444

# Checklist

- [ ] All acceptance criteria are met.
- [x] I've added or updated relevant docs in the docs/ directory.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: api/views/phones.py [hunk 1]
```diff
@@ -611,6 +613,87 @@ def sms_status(request):
...
+        )
+        outbound_messages = client.messages.list(
+            from_=relay_number.number, to=contact.inbound_number
+        )
+        return response.Response(
+            {
+                "inbound_messages": inbound_messages,
+                "outbound_messages": outbound_messages,
+            },
+            status=200,
```
Comment (codemist) [inline api/views/phones.py:683]: Could you split the messages into two parts instead of having it contain both `inbound_messages` and `outbound_messages`?

Comment (groovecoder) [reply api/views/phones.py:683]: How about a url parameter?

**Comment by groovecoder** [comment]
image

File: api/views/phones.py (hunk 1 above)
Comment (codemist) [reply api/views/phones.py:741]: Yeah that works too

**Comment by groovecoder** [comment]
Closing this PR. We can keep the branch and push it up to dev server as-needed for clients to work on it. We can also re-open this when we have work planned that needs new API endpoints.

**Pull request closed without merging**